library/Ivoz/Provider/Domain/Model/Voicemail/VoicemailDto.php

```diff
@@ -21,6 +23,21 @@ public static function getPropertyMap(string $context = '', string $role = null)
             ];
         }
 
+        if ($context === self::CONTEXT_MY_VOICEMAILS) {
```

Comment:
Use default context

Reply:
I made that change and it affects also the results from company_voicemails. It's only used from call_forward_settings and everything is still working fine.